Jetpacks Now Useable on suitStorage Slot

Sleepy-Shinu opened the pull request:
# Description
- Jetpacks such such as Black, Blue, and Security can now be on the suit storage slot.
- Moved all applicable textures to the dedicated _Floof resources folder. (For any future changes of looks/the likes).
- Added in applicable .png files to display textures of jetpacks.
- Modified metadata of each Jetpacks.rsi to accept new (old) textures for displaying.

# Balance changes
- Space fights now possible, as you will not have to drop your jetpack to fight / Get mini jetpacks to apply them to the suit storage slot. This allows sec/antags to use a gun and fly.
- Salvage has a reason to spend 1k to buy the larger storage blue jetpacks, purchasable at cargo computers.
-  Engineers, security, salvage, and tiders alike can now fly freely in space without worrying about noospheric storms or hunting mini jetpacks.

---

# TODO
- Allow the more artistically inclined to rework jetpack textures, if they so choose.
- Give Floof Stations LLC/NT Epistemics their payment for adding clamps to the various jetpacks. 

- [ ] Task
- [x] Completed Task

---
<details>
<p>
<img width="600" height="790" alt="image" src="https://github.com/user-attachments/assets/1f2ea194-5c39-4d16-b7c7-dbbd3e34c566" />

</p>
</details>

---

# Changelog

:cl: Ikiru-is-sleepless
- tweak: All jetpacks can now be used on the "suitstorage" slot. Happy flying.

Sleepy-Shinu committed:
Jetpacks now useable on suitStorage slot

- Jetpacks such such as Black, Blue, and Security can no be work on the suit storage slot.
- Moved all applicable textures to the dedicated _Floof resources folder. (For any future changes of looks/the likes).
- Added in applicable .png files to display textures of jetpacks.
- Modified metadata of each Jetpacks.rsi to accept new (old) textures for displaying.

Mnemotechnician reviewed (changes requested):
This shouldn't introduce duplicate states. If you want to use the same sprite for multiple slots, modify the appropriate field of ClothingComponent. See harnesses for a reference: https://github.com/Floof-Station/Floof-Station/blob/master/Resources/Prototypes/_Floof/Entities/Clothing/Uncategorized/harness.yml

Sleepy-Shinu commented:
> This shouldn't introduce duplicate states. If you want to use the same sprite for multiple slots, modify the appropriate field of ClothingComponent. See harnesses for a reference: https://github.com/Floof-Station/Floof-Station/blob/master/Resources/Prototypes/_Floof/Entities/Clothing/Uncategorized/harness.yml

Are you referring of the addition of the "XYZ_SUITSTORAGE" sprites including the metadata of the .rsi folder -or- within the Components<Clothing<Slots of jetpacks.yml? 

Could you point out a more direct example of duplicate state, I seem to not fully grasp which aspect you are highlighting -- Thank you much.

PS -- Fat fingered re-review. Clearly not needed yet.

Mnemotechnician commented:
> Are you referring of the addition of the "XYZ_SUITSTORAGE" sprites including the metadata of the .rsi folder -or- within the Components<Clothing<Slots of jetpacks.yml?
> 
> Could you point out a more direct example of duplicate state, I seem to not fully grasp which aspect you are highlighting -- Thank you much.

The states equipped-BACKPACK and equipped-SUITSTORAGE, as well as on-equipped-BACKPACK and on-equipped-SUITSTORAGE are duplicated in each RSI. This will cause issues later if someone decides to edit one of those jetpacks or decides to add a new one without repeating the same RSI state duplication, and will unnecessarily bloat the already bloated sprite atlas.

Sleepy-Shinu commented:
> > Are you referring of the addition of the "XYZ_SUITSTORAGE" sprites including the metadata of the .rsi folder -or- within the Components<Clothing<Slots of jetpacks.yml?
> > Could you point out a more direct example of duplicate state, I seem to not fully grasp which aspect you are highlighting -- Thank you much.
> 
> The states equipped-BACKPACK and equipped-SUITSTORAGE, as well as on-equipped-BACKPACK and on-equipped-SUITSTORAGE are duplicated in each RSI. This will cause issues later if someone decides to edit one of those jetpacks or decides to add a new one without repeating the same RSI state duplication, and will unnecessarily bloat the already bloated sprite atlas.

Okay perfect! I was just unsure what you quite meant-- I had copied the robust toolbox variation from "CaptainsJetpack.RSI." However I'll see if I cant modify the respective Jetpacks meta data to fix this!

Sleepy-Shinu committed:
Removed Bloat of multiple sprites

- Removed the Equipped-Suitstorage, Equipped-Backpack, etc sprites
- Combined all equipped sprites into an "equipped" tag
- Updated metadata / YML to reflect to use these tags.

Sleepy-Shinu committed:
Delete excess copied file.

Had a copy of meta.json that wasnt needed in captain.rsi

Mnemotechnician reviewed (commented):
Unmarked stale. Why does this still introduce new sprites instead of modifying the original though?
